dashboard/views/data_group.py

```diff
@@ -7,14 +7,16 @@
 from django.http import HttpResponse, JsonResponse
 from django.shortcuts import render, redirect, get_object_or_404
 from django.template.defaultfilters import pluralize
+from bulkformsets.serializers import CSVReader
 
-from dashboard.forms import DataGroupForm, create_detail_formset
+from dashboard.forms import DataGroupForm
```

Comment:
```suggestion
from dashboard.forms import DataGroupForm, create_detail_formset
```
The view `habitsandpractices()` depends on it.

Reply:
Committed this. Do we not have a test for catching that?

Reply:
It looks like we don't. I'm not sure what functionality this page adds that the DataDocument detail doesn't provide.